### PR TITLE
New build_style cargo

### PR DESCRIPTION
**Pull request by DirectorX**

### CODE REVIEW

**Comment by Vaelatern** [comment]
Can you put the whitespace fixes in their own PR?

**Comment by DirectorX** [comment]
@Vaelatern whitespace fixes removed

File: Manual.md [hunk 1]
```diff
@@ -650,6 +650,8 @@ be passed in via `make_build_args` and install arguments via `make_install_args`
 target can be overridden via `make_build_target` and the install target
 via `make_install_target`.
 
+`cargo` For rust packages that use cargo. Currently building rust packages available `only_for_archs="x86_64"`
```
Comment (Vaelatern) [inline Manual.md]: This is going to be fixed, the rust `only_for_archs`.

Comment (DirectorX) [inline Manual.md]: a remark added

**Comment by ebfe** [comment]
This doesn't work as intended, see #4526

**Comment by Vaelatern** [comment]
How does it not work as intended? Seeing the other PR did not clarify the matter at all.

**Comment by ebfe** [comment]
> How does it not work as intended? Seeing the other PR did not clarify the matter at all.
do_install not installing anything into ${DESTDIR}, seems pretty broken
to me...

**Comment by Vaelatern** [comment]
ty for the clarification

**Comment by DirectorX** [comment]
@Vaelatern @ebfe 
i didn't managed to make `cargo install` work. now the `do_install` hook is like the hook in `build-style-go`.

**Comment by the-maldridge** [comment]
@DirectorX Is this PR still active?

**Comment by DirectorX** [comment]
Not really, I need to to restart my work since cargo changed.

24. Jun 2017 02:59 by notifications@github.com:

**Pull request closed without merging**